src/huntsman/pocs/states/huntsman/initialising.py

```diff
@@ -0,0 +1,18 @@
+""" The goal of the initialising state is to prepare Huntsman for observing, including opening
+the dome and cooling the cameras. Note that it is the responsibility of the state machine to
+check it is safe before entering this (and all other) states. """
+
+
+def on_enter(event_data):
+    """
+    """
+    pocs = event_data.model
+    pocs.next_state = 'parking'
+    pocs.say("Initialising Huntsman.")
+
+    pocs.observatory.prepare_cameras()
+    pocs.observatory.mount.unpark()
+    pocs.observatory.open_dome()
+
+    pocs.say("Initialisation complete.")
+    pocs.next_state == "ready"  # If not safe, the state machine goes to park automatically
```

Comment:
Should handle exceptions, at a minimum a wide net like:

```suggestion
    try:
        pocs.observatory.prepare_cameras()
        pocs.observatory.mount.unpark()
        pocs.observatory.open_dome()

        pocs.say("Initialisation complete.")
        pocs.next_state == "ready"  # If not safe, the state machine goes to park automatically
    except Exception as e:
        pocs.logger.warning(f'Cannot initialise {e!r}')
```

Reply:
@wtgee I think unhandled state errors should (ideally) be handled by the state machine, no?